src/components/sectioned-navigation/sectioned-navigation.js

```diff
@@ -16,12 +24,55 @@ class SectionedNavigation extends React.Component {
     );
   }
 
+  renderFilterBar() {
+    const { props } = this;
+    if (!props.includeFilterBar) {
+      return null;
+    }
+    return (
+      <div className="wmax360 mb24">
+        <input
+          onChange={e =>
+            this.setState({ filter: e.target.value }, this.filterResults())
```

Comment:
I think you'll want `this.filterResults` instead of `this.filterResults()`. The difference is that `this.filterResults()` would be called as soon as `onChange` is called (because of the `()`), but `this.filterResults` would be referencing the function that would then be called as the callback to `this.setState`.